ql/src/test/queries/clientpositive/order_null2.q

```diff
@@ -0,0 +1,70 @@
+create table test1
+(
+  a string,
+  b timestamp,
+  c timestamp
+);
+
+INSERT INTO TABLE test1 VALUES
+('John Doe', '1990-05-10 00:00:00.0', '2022-01-10 00:00:00.0'),
+('John Doe', '1990-05-10 00:00:00.0', '2021-12-10 00:00:00.0'),
+('John Doe', '1990-05-10 00:00:00.0', '2021-11-10 00:00:00.0'),
+('John Doe', '1990-05-10 00:00:00.0', '2021-10-10 00:00:00.0'),
+('John Doe', '1990-05-10 00:00:00.0', '2021-09-10 00:00:00.0'),
+('John Doe', '1987-05-10 00:00:00.0', '2022-01-10 00:00:00.0'),
+('John Doe', '1987-05-10 00:00:00.0', '2021-12-10 00:00:00.0'),
+('John Doe', '1987-05-10 00:00:00.0', '2021-11-10 00:00:00.0'),
+('John Doe', '1987-05-10 00:00:00.0', '2021-10-10 00:00:00.0'),
+('John Doe', '1987-05-10 00:00:00.0', null);
```

Comment:
I suppose the example can be simplified a bit by: 

1. picking more user-friendly names/types; 
2. using fewer rows; 

```
create table person (name string, country string, age int);
insert into person values 
('John Doe', 'France', 20),
('Alex Dum', 'France', 24),
('Victor Hug', 'France', null);
```